Tools/auterion/upload_skynode.sh

```diff
@@ -29,8 +37,15 @@ done
 [ -z "$AUTOPILOT_PORT" ] && AUTOPILOT_PORT=$DEFAULT_AUTOPILOT_PORT
 [ -z "$AUTOPILOT_USER" ] && AUTOPILOT_USER=$DEFAULT_AUTOPILOT_USER
 
-echo "Uploading to $AUTOPILOT_HOST..."
+ARGUMENTS=""
```

Comment:
Can you use an array to handle spaces in the path correctly?
```
array=()
array+=(xy)
```

Then pass it as `"${array[@]}"`

Reply:
Thanks for the hint. I fighted with the spaces and didn't know about this possibility. Done in cdc784e
When echoing commands it still looks correct.